internal/sub_reconcilers/reconcile_deployment_size.go

```diff
@@ -60,6 +62,11 @@ func (r *DeploymentSizeReconciler) Reconcile(ctx context.Context, bestie *petsv1
 	if err != nil {
 		r.Log.Error(err, "unable to retrieve deployment")
 	}
+	// Validate that size <= MaxReplicas
+	if bestie.Spec.Size > *bestie.Spec.MaxReplicas {
```

Comment:
This is confusing to me to read, the comment says `less than or equal to` but the code says `greater than`. Unless I don't know how to read which is quite possible. Maybe all english would better for the comment.

Reply:
@skattoju , Could you please rewrite the comment?

Reply:
maths! but ok will un-reverse 😅